.gitmodules

```diff
@@ -20,3 +20,15 @@
 	path = playbooks/roles/load-balancer
 	url = https://github.com/open-craft/ansible-load-balancer
 	branch = master
+[submodule "playbooks/roles/geerlingguy.nginx"]
```

Comment:
@Agrendalath, can we not use the existing `nginx` role in this repository and avoid using a separate, duplicate sub-module?